tests/unit/test_cli.py

```diff
@@ -926,3 +968,23 @@ def test_a_user_can_successfully_save_sboms_merge_results_to_a_file(
             fp.close()
             invoke_expecting_sucess(app, ["merge-sboms", "-o", fp.name, *sbom_files_to_merge])
             assert Path(fp.name).lstat().st_size > 0, "SBOM failed to be written to output file!"
+
+    @pytest.mark.parametrize(
+        "sbom_files_to_merge",
+        [
+            [
+                "./tests/unit/data/sboms/cachi2.bom.spdx.json",
+                "./tests/unit/data/sboms/syft.bom.spdx.json",
```

Comment:
Nitpick: The `merge-sboms` command was meant to only be used with Cachi2-produced SBOMs. There's no guarantee that merging a Syft SBOM would work. So I think the test data here ends up being misleading.

Reply:
It's syft-convert of ./tests/unit/data/sboms/cachi2.bom.spdx.json